frontend/packages/console-shared/src/components/markdown-highlight-extension/highlight-consts.ts

```diff
@@ -0,0 +1,10 @@
+export const LINK_LABEL = '[\\d\\w\\s-()$!]+';
+export const HIGHLIGHT_ACTIONS = ['highlight'];
+const ID_PREFIX = '(qs-|tour-)';
```

Comment:
I don't think we need to limit IDs to a specific prefix considering we only look at a specific data attribute.

Reply:
done